frontend/src/components/common/authorized-wrapper/styles.module.scss

```diff
@@ -1,4 +1,9 @@
 .content {
   display: flex;
   flex-grow: 1;
+  min-height: 0;
+}
+
+.mainContent {
+  overflow: auto;
 }
```

Comment:
You need to replace "overflow: auto;" with "overflow-y: auto;" because ther is no need for us in scroll on horizontal axis